mgc/discrim/discrimOneSample.py

```diff
@@ -2,12 +2,12 @@
 from ._utils import _CheckInputs
 import numpy as np
 import random
-from .base import discriminabilityTest
+from .base import DiscriminabilityTest
 from scipy._lib._util import MapWrapper
 
 
 
-class oneSample(discriminabilityTest):
+class DiscrimOneSample(DiscriminabilityTest):
     r"""
      A class that performs a one-sample test for whether the discriminability 
      differs from random chance, as described in [1]. Discriminability index 
```

Comment:
again, math should be in a separate notes section. Also, reference should be '[#1Dscr]_'